src/lang/std/engineConnection.ts

```diff
@@ -16,6 +16,8 @@ import { useModelingContext } from 'hooks/useModelingContext'
 import { exportMake } from 'lib/exportMake'
 import toast from 'react-hot-toast'
 import { SettingsViaQueryString } from 'lib/settings/settingsTypes'
+import { kclManager } from 'lib/singletons'
```

Comment:
I suspect this is the problem, importing from the singletons module into one of the singletons.

Reply:
Okay I've push ab13c25345b204cdefac2277e73fe3c78aa7aabd, hopefully build-test-web is happy now

Reply:
Thanks for catching this and fixing it. I appreciate it. I'll take a look into the import pattern you used.